xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-transformations/xwiki-platform-rendering-transformation-icon/src/main/resources/META-INF/components.txt

```diff
@@ -1 +1,2 @@
 500:org.xwiki.rendering.internal.transformation.icon.XWikiIconTransformationConfiguration
+org.xwiki.rendering.internal.transformation.icon.XWikiIconProvider
```

Comment:
Don't you also need a `500:` here to safely override the other implementation?

Reply:
Sure :+1:  I wasn't sure wether to put it or not. When testing out my changes, it worked without it, and I figured I shouldn't add it if it's unnecessary.
I didn't find the [XWiki component documentation](https://extensions.xwiki.org/xwiki/bin/view/Extension/Component%20Module#HOverrides) when coding this part and it was a bit fuzzy still...